UefiTestingPkg/AuditTests/BootAuditTest/UEFI/BootAuditTestApp.inf

```diff
@@ -37,4 +37,12 @@
   UefiApplicationEntryPoint
   UefiLib
   UefiRuntimeServicesTableLib
-  UnitTestLib
\ No newline at end of file
+  UnitTestLib
+  DevicePathLib

```

Comment:
The library class items were sorted before. Could you sort these with the existing items?

Reply:
Sorted